aea/test_tools/test_cases.py

```diff
@@ -712,7 +715,7 @@ def setup_class(cls):
         """Set up the test class."""
         # make paths absolute
         cls.path_to_aea = cls.path_to_aea.absolute()
-        cls.packages_dir_path = cls.packages_dir_path.absolute()
+        # TODO: decide whether to keep optionally: cls.packages_dir_path = cls.packages_dir_path.absolute()
```

Comment:
why not just `None` if `use_packages_dir = False`? It's responsibility of the user to know whether he can use it or not.

Reply:
Well there is already quite a lot for the user to know to use this test case anyway. We need to improve them, but the quick fix is as I did so not to run into any edge cases where an optional field might cause issues.